src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  gap: 48px 46px;
+  grid-template-columns: repeat(var(--column-count), 1fr);
+  justify-items: center;
+
+  @include screen-xs {
+    --column-count: 2;
+  }
+
+  @include screen-sm {
+    --column-count: 3;
+  }
+
+  @include screen-md {
```

Comment:
Ensure that the mixin `screen-md` is defined and correctly imported in your SCSS setup to prevent any compilation errors.